WorkflowAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/WorkflowProfile/WorkflowProfileFormView.js

```diff
@@ -1,15 +1,32 @@
-import AbstractFormView from '../../../Service/Form/View/AbstractFormView'
+import AbstractFormView     from '../../../Service/Form/View/AbstractFormView'
+import Application          from '../../Application'
+import WorkflowProfile      from '../../Model/WorkflowProfile/WorkflowProfile'
+import FormViewButtonsMixin from '../../../Service/Form/Mixin/FormViewButtonsMixin'
 
 /**
  * @class WorkflowProfileFormView
  */
-class WorkflowProfileFormView extends AbstractFormView
+class WorkflowProfileFormView extends mix(AbstractFormView).with(FormViewButtonsMixin)
 {
     /**
+     * Initialize
+     * @param {Form}   form
+     * @param {String} workflowProfileId
+     * @param {Array}  name
```

Comment:
name is Translator.trans('open_orchestra_workflow_admin.workflow_profile.title_new') (line 66 in WorkflowProfileRouter)